.github/workflows/update-codeclimate-coverage.yml

```diff
@@ -0,0 +1,27 @@
+# Summary:
+# Run Unit and coverage tests, then upload it to Code Climate dashboard
+# Read ./README.md for extensive documentation
+
+name: Update Code Climate Coverage
+
+on:
+  push:
+    branches:
```

Comment:
Doc: Specify this must be the branch defined in CC as "default" branch

![image](https://user-images.githubusercontent.com/3807458/84020452-1cf74b00-a983-11ea-9e07-f8ee7ead6d78.png)